### ISSUE TITLE
[a11y] Tabs

### ISSUE THREAD

**Issue by GrandSchtroumpf** [bancorprotocol/carbon-app#820]
## Tabs
Use tab when: 
- multiple panels are the same level
- only one panel should be visible
- panels should be a sub section of the page, but not a page

Do not use menu for : 
- tabs should not update URL -> use nav
- tabs should not select option in a form -> use radio

Attributes: 
- `role`: `tablist`, `tab`, `tabpanel`
- `aria-selected`: `true` for the selectd tabs
- `aria-controls`: id of the panel
- `aria-labelledby`: tabpanel should be labelled by their tab
- `tabIndex`: `tab` -> `0` is selected, `-1` else /  `tabpanel` -> 0

Note: navigation should behave like that : 
- Arrow to navigate between tabs
- `Tab` to navigate between tab & panel

```html
<div role="tablist">
  <button id="depthTabId" role="tab" type="button" aria-selected="true" aria-controls="depthPanelId">
    Depth
  </button>
  <button id="priceTabId" role="tab" type="button" aria-selected="false" aria-controls="pricePanelId">
    Price
  </button>
</div>
<article role="tabpanel" id="depthPanelId" aria-labelledby"depthTabId">
</article>
<article role="tabpanel" id="pricePanelId" aria-labelledby="priceTabId" hidden>
</article>
```

**Keyboard navigation** on `tablist`
- `ArrowRight` / `ArrowLeft`: focus next tab
- `Enter` / ` ` (space): Select tab (change `aria-selected`, `tabIndex` & remove `hidden` on the panel)
As an alternative, `ArrowRight`/`ArrowLeft` can also select the tab.

Use it :
- [ ] Depth/Price in Trade page
- [ ] Buy / Sell in trade page on mobile
